src/main/java/nl/hr/recipefinder/config/SecurityConfig.java

```diff
@@ -60,7 +60,10 @@ protected void configure(HttpSecurity http) throws Exception {
         "/admin/**",
         "/swagger-ui/**",
         "/h2-console/**"
-      ).hasRole(Role.ADMIN.name())
+      ).hasAnyRole(Role.ADMIN.name(), Role.USER.name())
+      .antMatchers(
+        "/report/**").
+      hasRole(Role.USER.name())
```

Comment:
Hold on, you nearly set the `admin` security settings to allow users with role `USER`

```suggestion
      ).hasRole(Role.USER.name())
      .antMatchers(
        "/report/**"
       ).hasAnyRole(Role.ADMIN.name(), Role.USER.name())
```